### ISSUE TITLE
Only Mine setting not working

### ISSUE THREAD

**Issue by Mrakulos** [Bodify/BetterBlizzPlates#13]
Hello, I'd like to bring up an issue I think I'm having with the addon which is that "Only mine" setting in Nameplate Auras doesn't work. Especially for debuffs, it shows everyone's debuffs (in the screenshot you can see a Rogue's Garote CC, but I'm a Warlock). I'm also not sure if the "Blizzard Default Filter" setting is working as it shows other auras that normally do not show up with Blizzard Default setting. Thanks for looking into this if you can

![image](https://github.com/user-attachments/assets/9ee671bc-115d-4891-9982-dadd040efa38)
![image](https://github.com/user-attachments/assets/abab874e-f2cb-4f73-a01b-0a1e97bdd57f)

**Comment by Bodify** [comment]
Hey! It is mentioned in the tooltip that "Only mine" allows auras from the Blizzard Default Filter so it is working as intended there. As for it showing auras not normally shown that does not sound accurate as I use the original Blizzard function with their filter. 
Can you mention an aura that comes through the default filter that isnt yours, because of "Only Mine"?

**Comment by Mrakulos** [comment]
Example 1 (in my original screenshots): it's **not** showing only mine, because you can see 2 debuffs on the target dummy. 1 debuff is from me (warlock that applied corruption). Debuff 2 is from a random Rogue that applied Garrote, you can see it in the tooltip at the top

Example 2 (new screenshot): I am a warlock that is applying "Drain Soul" to the target dummy. Drain Soul debuff is **not** part of the default blizzard filters

![image](https://github.com/user-attachments/assets/e94fdc4c-2985-4c9e-93af-05b7f60ee6c0)

**Comment by Bodify** [comment]
Hello again. I think you're confused. I have already explained that the "Default Blizzard Filter" allows certain auras from other classes, like garrote as you've mentioned.

The "Only Mine" setting specifically says in the tooltip it DOES allow these auras from the default filter to go through and conflict the "Only Mine" setting, this is done intentionally. You should turn off the default filter if you want to change this.

The reason you are seeing Drain Soul is because you have "Only Mine" selected and the drain soul debuff is yours therefore it shows.
The "Default Blizzard Filter" and "Only Mine" filter together would not make any sense to have on together otherwise as the default filter would just block out any additional ones from "Only Mine that is not part of the default filter.

If you do not want to show Drain Soul just add it to the blacklist.

**Comment by Mrakulos** [comment]
Thanks for explaining that, the logic makes sense, but the use case doesn't. I feel like a popular request would be able to use the vanilla blizzard aura settings -- where it only shows your own debuffs + the ones that blizzard filters by default (which instinctively would be only mine + default blizzard filter no?). The use case is being able to use the addon but have the vanilla blizzard aura setup + adding pandemic glow to haunt/agony

**Comment by Bodify** [comment]
No that is incorrect. In order to keep it like how the default nameplates are you should only have Blizzard Default Filter enabled. It is named Blizzard Default Filter because it literally is that; the default filter.
Adding extra filters will no longer have it be default. "Only Mine" will add extra auras that are not part of the default filter, such as Drain Soul for you.

You can add things to the whitelist for pandemic, important etc if you want and those will apply to the auras shown. You can read more about how it works on the whitelist tooltip.

**Comment by Mrakulos** [comment]
Hmm, I might either still be confused or there is a problem. Here is last example before I close thread.

Please see this example where Warrior did slaughterhouse, but I only have Blizzard Default Filter applied. Should I be seeing the other player's debuff?

![image](https://github.com/user-attachments/assets/602bd491-aa57-4508-9205-7b0139b20816)
![image](https://github.com/user-attachments/assets/0fcbfc8f-93d4-4727-972c-75278efeda63)

**Comment by Bodify** [comment]
Yes for the 4th and final time you should be seeing slaughterhouse, garrote and many other auras from other players that are part of the default blizzard filter. Turn off all addons and you will still see it, that is how it is by default. You can avoid this by unchecking it and turning on "Only Mine" instead but doing so you would have to blacklist some that the default filter used to handle, like Drain Soul for example.
